src/sync.rs

```diff
@@ -460,7 +484,7 @@ UNLISTED 🔓 ✅ Tagged people
 
         let tweets = vec![tweet];
         let statuses = vec![status];
-        let posts = determine_posts(&statuses, &tweets);
+        let posts = determine_posts(&statuses, &tweets, &DEFAULT_SYNC_OPTIONS);
```

Comment:
this is a bit annoying that we have this API change. I thought about doing a bit of compatibility dance and keep determine_posts() with the same function signature. Then add determine_posts_with_config() which can be called by main(). But probably that is not worth it and just adds one layer of indirection, and refactoring tests like this is fine. mastodon-twitter-sync is not a library, so we can mess up function signatures all we want :)

Reply:
Yeah, it is annoying. On the other hand, now that we have made this change, adding more sync options should be much easier in the future.